deployment/packer/mmw.json

```diff
@@ -81,28 +81,6 @@
                 "Environment": "{{user `stack_type`}}"
             },
             "associate_public_ip_address": true
-        },
```

Comment:
I recall you mentioning that you wanted to convert the Packer builder to the T3 instance family.

Reply:
Thanks for surfacing this.